tests/unit/test_coco_validation.py

```
@@ -46,6 +47,11 @@ def compute(self, save_json=None):
         return dict(self.metrics)
 
 
+def _null_val_preprocessor(*, img_size):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_null_val_preprocessor`

Add return type annotation: `None`

(ANN202)